src/util/MembershipQueue.ts

```diff
@@ -15,7 +15,7 @@ interface QueueUserItem {
     reason?: string;
     attempts: number;
     roomId: string;
-    userId: string;
+    userId?: string;
```

Comment:
Marking LGTM, but can we talk about why userId is now optional throughout the Queue?